matrix_sdk/Cargo.toml

```diff
@@ -20,12 +20,12 @@ default = ["encryption", "sled_cryptostore", "sled_state_store", "native-tls"]
 encryption = ["matrix-sdk-base/encryption"]
 sled_state_store = ["matrix-sdk-base/sled_state_store"]
 sled_cryptostore = ["matrix-sdk-base/sled_cryptostore"]
-unstable-synapse-quirks = ["matrix-sdk-base/unstable-synapse-quirks"]
 markdown = ["matrix-sdk-base/markdown"]
 native-tls = ["reqwest/native-tls"]
 rustls-tls = ["reqwest/rustls-tls"]
 socks = ["reqwest/socks"]
 sso_login = ["warp", "rand", "tokio-stream"]
+require_auth_for_profile_requests = []
```

Comment:
Do we want to enable this by default? I think as soon as someone starts using the SDK with the Gnome homeserver, they'll be getting in trouble with this.

Reply:
Given that it just sends an authorization header I'd say it doesn't hurt to enable by default. Will change it.

I'm curious, there's a Gnome homeserver in development? Where can one read about that?

Reply:
I think the gnome homeserver is simply a (large) Synapse instance that enables `require_auth_for_profile_requests`.

Reply:
That's correct, I mean the homeserver that the Gnome community uses, I think it's hosted by EMS.

Reply:
@jplatte Ah, that explains it, thanks!